CMakeLists.txt

```diff
@@ -106,11 +108,11 @@ function(finufft_link_test target)
           endif()
       endif()
   else()
-      target_link_libraries(${target} PRIVATE finufft)
       if(WIN32)
           target_compile_definitions(${target} PRIVATE FINUFFT_DLL)
       endif()
   endif()
+  target_link_libraries(${target} PRIVATE finufft)
```

Comment:
this line was in the else branch of `if (FINUFFT_STATIC_LINKING)` for dynamic linking, I guess we still want to keep only static linking if FINUFFT_STATIC_LINKING is ON.